server/core/src/https/views/errors.rs

```diff
@@ -21,27 +21,31 @@ use crate::https::middleware::KOpId;
 pub(crate) enum HtmxError {
     /// Something went wrong when doing things.
     OperationError(Uuid, OperationError),
-    // InternalServerError(Uuid, String),
+    InternalServerError(Uuid, String),
 }
 
 impl HtmxError {
     pub(crate) fn new(kopid: &KOpId, operr: OperationError) -> Self {
         HtmxError::OperationError(kopid.eventid, operr)
     }
+
+    pub(crate) fn internal(kopid: &KOpId, msg: String) -> Self {
+        HtmxError::InternalServerError(kopid.eventid, msg)
+    }
 }
 
 impl IntoResponse for HtmxError {
     fn into_response(self) -> Response {
         match self {
-            // HtmxError::InternalServerError(_kopid, inner) => {
-            //     (StatusCode::INTERNAL_SERVER_ERROR, inner).into_response()
-            // }
+            HtmxError::InternalServerError(_kopid, inner) => {
```

Comment:
Are we sure we're returning the `kopid` as part of these responses? Dropping the `kopid` makes it harder to troubleshoot.